modules/jsonrpc/jsonrpc.cpp

```diff
@@ -98,6 +98,10 @@ Variant JSONRPC::process_action(const Variant &p_action, bool p_process_arr_elem
 	if (p_action.get_type() == Variant::DICTIONARY) {
 		Dictionary dict = p_action;
 		String method = dict.get("method", "");
+		if (method.find("$/") == 0) {
```

Comment:
Shouldn't this be `if (method.begins_with("$/")) {`?

Reply:
It should give the same result, but yes `begins_with` is cleaner and possibly faster.

Reply:
Ahhh. Keeping track of all the string manipulation functions is a challenge. I'll edit.